go.mod

```diff
@@ -103,4 +87,6 @@ require (
 
 replace github.com/jensneuse/graphql-go-tools => github.com/TykTechnologies/graphql-go-tools v1.6.2-0.20220509111320-96782935e66c
 
+replace gorm.io/gorm => github.com/TykTechnologies/gorm v1.20.7-0.20210409171139-b5c340f85ed0
```

Comment:
Can you verify this one? I thought I removed the `_` import from apidef, gorm is not used here.